python/pyhail/dataset.py

```diff
@@ -830,13 +1081,43 @@ def hardcalls(self):
     def ibd(self, output, maf=None, unbounded=False, min=None, max=None):
         """Compute matrix of identity-by-descent estimations.
 
+        **Examples**
+
+        Suppose we have a variant annotation ``va.panel_maf`` of allele
+        frequencies computed from a reference panel. Then
+
+        >>> (hc.read('example.vds')
+        >>>  .ibd('ibd.tsv', maf='va.panel_maf', min=0.2, max=0.9)
+
+        outputs to ``ibd.tsv`` only those sample pairs with ``pi_hat`` between 0.2 and 0.9 inclusive, using the the provided expression for the minor allele frequency.
+
+        To writes the full IBD matrix to `ibd.tsv` with the minor allele frequency computed from the dataset:
+
+        >>> (hc.read('example.vds')
+        >>>  .ibd('ibd.tsv'
+
+        **Notes**
+
+        The implementation is based on the IBD algorithm described in the `PLINK paper <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838>`_.
+
+        ``ibd`` requires the dataset to be bi-allelic (otherwise run ``split_multi``). ``ibd`` does not perform LD
```

Comment:
Refer to split_multi and ibd like this:

:py:meth:\`~pyhail.VariantDataset.split_multi\`

:py:meth:\`~pyhail.VariantDataset.ibd\`